controller/handler/trial.js

```diff
@@ -11,6 +11,7 @@ const processComplianceCount = require('../helper/process-compliance-count');
 const processRules = require('../helper/process-rules');
 const processPatientStatus = require('../helper/process-patient-status');
 const httpNotFound = 404;
+const sqlDateFormat = 'ddd MMM DD YYYY HH:mm:ss ZZ';
```

Comment:
can you explain why do you require this date format?
Later at line 125 and 126, you do have a format applied to the date to get it in the required format?